.ado/jobs/jschecks.yml

```diff
@@ -11,17 +18,29 @@ jobs:
     steps:
       - checkout: self # self represents the repo where the initial Pipelines YAML file was found
         clean: true # whether to fetch clean each time
-        fetchDepth: 2 # the depth of commits to ask Git to fetch
         lfs: false # whether to download Git-LFS files
         submodules: false # set to 'true' for a single level of submodules or 'recursive' to get submodules of submodules
         persistCredentials: false # set to 'true' to leave the OAuth token in the Git config after the initial fetch
 
       - template: ../templates/yarn-install.yml
 
+      - script: |
```

Comment:
Could you do something like below?

```yaml
variables:
  - BeachBallBranchName: coalesce(System.PullRequest.SourceBranchName, Build.SourceBranch)
```

Not actual syntax, but could avoid dynamic variable setting steps

Reply:
investigating, good idea